### ISSUE TITLE
Error: [libs/sql-schema-describer/src/sqlite.rs:452:76] get name

### ISSUE THREAD

**Issue by jonnyparris** [prisma/prisma#6499]
Hi Prisma Team! Prisma Migrate just crashed. 

## Command

`introspect`

## Versions
      
| Name        | Version            |
|-------------|--------------------|
| Platform    | darwin             | 
| Node        | v15.8.0            | 
| Prisma CLI  | 2.20.1             | 
| Binary      | 60ba6551f29b17d7d6ce479e5733c70d9c00860e| 

## Error
```
Error: [libs/sql-schema-describer/src/sqlite.rs:452:76] get name
```

**Comment by jonnyparris** [comment]
...trying to introspect an sqlite db

**Comment by janpio** [comment]
Did it prompt you to submit a bug report? If so, did you?
Can you share your database schema with us?

**Comment by jonnyparris** [comment]
Yup, bug report prompted and submitted.
What's the best way to share just the schema?

**Comment by jonnyparris** [comment]
Unless you meant the `prisma.schema`(?) which is as follows:
```
// This is your Prisma schema file,
// learn more about it in the docs: https://pris.ly/d/prisma-schema

datasource db {
  provider = "sqlite"
  url      = env("DATABASE_URL")
}

generator client {
  provider = "prisma-client-js"
}
```

**Comment by janpio** [comment]
I found the connected error reports: 13207 and 13208. We can use that internally to try to find out what is going on.

If you are comfortable sharing the schema publicly, just post it here as a comment or via a Gist.
If it needs to stay private, send it via email to schemas@prisma.io and include a link to this issue so we can make the connection.

**Comment by jonnyparris** [comment]
Here's a [gist](https://gist.github.com/jonnyparris/f24a82dc75801691ab79d0bb997affb7)

**Comment by pantharshit00** [comment]
I can reproduce this with the database dump. Thanks for taking the time and sharing the dump with us 

![image](https://user-images.githubusercontent.com/22195362/114386813-55714200-9baf-11eb-9658-bb6cf832ba93.png)

**Comment by jonnyparris** [comment]
is there any way to use this fix before the 2.22 milestone gets released? 🙏🏾

**Comment by jonnyparris** [comment]
amazing, thanks so much for the rapid replies and rapid turnaround!
